src/Application/Network/Queries/GetNetworkProfileList/GetNetworkProfileListQuery.cs

```diff
@@ -0,0 +1,94 @@
+using AutoMapper.QueryableExtensions;
+using SSW.Rewards.Shared.DTOs.Leaderboard;
+using SSW.Rewards.Shared.DTOs.Network;
+using SSW.Rewards.Shared.DTOs.Users;
+
+namespace SSW.Rewards.Application.Network.Queries;
+
+public class GetNetworkProfileListQuery : IRequest<NetworkProfileListViewModel>;
+
+public class GetNetworkProfileListHandler : IRequestHandler<GetNetworkProfileListQuery, NetworkProfileListViewModel>
+{
+    private readonly IApplicationDbContext _dbContext;
+    private readonly IUserService _userService;
+    private readonly IMapper _mapper;
+
+    public GetNetworkProfileListHandler(
+        IApplicationDbContext dbContext, 
+        IUserService userService, 
+        IMapper mapper)
+    {
+        _dbContext = dbContext;
+        _userService = userService;
+        _mapper = mapper;
+    }
+    
+    public async Task<NetworkProfileListViewModel> Handle(GetNetworkProfileListQuery request, CancellationToken cancellationToken)
+    {
+        var profiles = new List<NetworkingProfileDto>(200);
+        var user = await _userService.GetCurrentUser(cancellationToken);
+        var achievements = await _userService.GetUserAchievements(user.Id, cancellationToken);
+        
+        var completedAchievements = achievements.UserAchievements
+            .Where(a => a.Complete)
+            .Select(a => a.AchievementId)
+            .ToList();
+
+        var staffDtos = await _dbContext.StaffMembers
+            .Join(_dbContext.Users,
+                staff => staff.Email, 
+                user => user.Email,
+                (staff, user) =>
+                new
+                {
+                    UserId = user.Id,
```

Comment:
Why aren't you using an AutoMapper projection for this?

Reply:
Hi Matt,
I would not recommend using Automapper here, for example `user.Avatar ?? "v2sophie"`. This is logic that should not be in a mapping profile. Automapper makes it very difficult to debug, and hard to reason with the code. I don't think I could ever recommend using Automapper.

From Matt P

Reply:
Isn't that logic to display `v2sophie` if `Avatar` is null or empty handled in the UI? If not it should be.

Reply:
Hey Matt,

As per our conversation, we agree that the existing approach isn't the best, but might require Schema changes in order to work #759